app/videobridge/client/views/videoFlexTab.js

```diff
@@ -110,7 +110,14 @@ Template.videoFlexTab.onRendered(function() {
 			}
 
 			const domain = settings.get('Jitsi_Domain');
-			const jitsiRoom = settings.get('Jitsi_URL_Room_Prefix') + settings.get('uniqueID') + rid + settings.get('Jitsi_URL_Room_Suffix');
+			let rname;
+			if (settings.get('Jitsi_URL_Room_Hash')) {
```

Comment:
isn't this backwards?  If the setting is enabled you want to hash right?

Reply:
@geekgonecrazy If it's enabled, we want the hash yes; the hash is actually that `settings.get('uniqueID') + rid` so it's fine.

Reply:
Right.. but it’s working in reverse.  If hash set it’s not hashing.. and if hash is false then it is hashing.

Reply:
@geekgonecrazy  No it's not working in reverse. As said, the hash is actually produced by this `settings.get('uniqueID') + rid`. Give it a try and you'll see.
Nonetheless it made me try it again just to make sure I was not going crazy and I noticed a bug introduced by https://github.com/RocketChat/Rocket.Chat/commit/7db530c259#diff-6056610957cb6ad7c1d79a04eb6fc424R133 : in case jwt isn't enabled it adds "false" at the end of the jitsi url (in place of `queryString`) so I added this commit https://github.com/RocketChat/Rocket.Chat/pull/17481/commits/4622d66ca1dbf7fcb2ab628b4e702c2fc55920a3 to this PR in order to fix it. At least time was not wasted!

Reply:
Meanwhile https://github.com/RocketChat/Rocket.Chat/pull/18430 fixed that `false` bug as well (using another way), my last commit was in conflict, so I rebased on top of `develop` and removed my last commit. Would be happy to get another review @geekgonecrazy  :) thank you!

Reply:
Please @geekgonecrazy or maybe @rodrigok ? or even @engelgabriel ? maybe one of you can review this little piece of code ? :) thank you so much!

Reply:
Ah.. I think I misunderstood intent.  I thought this was making the feature more obscure but you are suggesting making an option to not obscure

Reply:
@geekgonecrazy  yes :) any blocker for approval ? :)